backend/src/api/routes/advisor/index.routes.ts

```
@@ -0,0 +1,75 @@
+
+const router = Router();
+const advisorService = DatabaseAdvisorService.getInstance();
+
+/**
+ * Trigger a database advisor scan.
+ * POST /api/advisor/scan
+ */
+router.post(
+  '/scan',
+  verifyAdmin,
+  async (_req: AuthRequest, res: Response, next: NextFunction) => {
```

ESLint

[error] Replace `⏎··'/scan',⏎··verifyAdmin,⏎·` with `'/scan',·verifyAdmin,`

(prettier/prettier)